### DISCUSSION TITLE
Decompile classes

### DISCUSSION THREAD

**Discussion by Matrixmobi** [REAndroid/APKEditor#32] [Q&A] [answered]
Hello. I am extremely impressed with this tool when using the merge feature. When decompile, I don't see the smali files. Are you planning to add decompile classes feature?

**Comment by REAndroid** [comment, marked as answer]
Some peoples requesting this feature lately and there is also good PR #24 , but there are unfinished priority features on this project. If  more requests are coming, I may prioritize it.

**Reply by SZRabinowitz** [reply]
k thanks for your amazing tool

**Comment by REAndroid** [comment]
Now fully implemented ( as of  [21d8efe](https://github.com/REAndroid/APKEditor/commit/21d8efee2cdd9e38ad9b828a569b41f629b47c56) ), check the latest update
